plaso/analysis/browser_search.py

```diff
@@ -175,6 +177,28 @@ def _ExtractGoogleSearchQuery(self, url):
 
     return line.replace('+', ' ')
 
+  def _ExtractYahooSearchQuery(self, url):
+    """Extracts a search query from a Yahoo search URL.
+
+    Yahoo: https://search.yahoo.com/search?p=query
```

Comment:
Please remove/reword 'Yahoo: ' I have no idea what you're telling me with this.

Maybe:
```
Examples:
  https://search.yahoo.com/search?p=query
  ...
```